### PR TITLE
Feat/challenge 03 loops conditionals

### PR DESCRIPTION
**Pull request by Enma1920**

### CODE REVIEW

File: challenge_03_loops_conditionals/README.md
Comment (tictools) [inline challenge_03_loops_conditionals/README.md:1]: `README.md` should live at project root level.

⚠️ FIX: move file at root level

File: challenge_03_loops_conditionals/loops-conditionals/.DS_Store
Comment (tictools) [inline challenge_03_loops_conditionals/loops-conditionals/.DS_Store:1]: ⚠️ FIX: remove file and add `.DS_Store` to `.gitignore`

File: challenge_03_loops_conditionals/loops-conditionals/conditionals/exercise_00.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    3. Create an if statement that checks if animal is equal to "cat"
+    4. If it is equal to "cat", print out "Meow!" to the screen
+    5. If it is equal to "dog", print out "Woof!" to the screen
+    6. If it is neither "cat" nor "dog", print out "Must be an alien"
+    7. At the end of the program print out: "--- Animal check complete ---"
+    8. Change animal to different values to make sure it works in all cases
+*/
+console.log("--- Animal Checker Program ---");
+let animal = "cat";
+if (animal == "cat") {
```
Comment (tictools) [inline challenge_03_loops_conditionals/loops-conditionals/conditionals/exercise_00.js]: ⚠️ FIX: Change animal to different values to make sure it works in all cases

File: challenge_03_loops_conditionals/loops-conditionals/conditionals/exercise_01.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    Create a program that starts with a number variable.
+    The program will return the variable is even or odd.
+
+    Change the variable to test it works for different cases.
+    Print result.
+
+    HINT: modulus operator ( % ) is your friend
+*/
+
+function Even_Odd(number) {
```
Comment (tictools) [inline challenge_03_loops_conditionals/loops-conditionals/conditionals/exercise_01.js]: Variables are always defined using camelCase convention, they never start by capital letter.

```suggestion
function evenOdd(number) {
```

⚠️ FIX: rename function

File: challenge_03_loops_conditionals/loops-conditionals/conditionals/exercise_01.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  if (number % 2 == 0) {
+    console.log(number + " is even");
+  } else {
+    console.log(number + " is odd");
+  }
+}
+
+console.log(Even_Odd(42));
```
Comment (tictools) [inline challenge_03_loops_conditionals/loops-conditionals/conditionals/exercise_01.js]: Your script's output is this one:

```
42 is even
undefined
5 is odd
undefined
```

Do you know why? 🤔

⚠️ FIX: spot the 🐛 and fix it

File: challenge_03_loops_conditionals/loops-conditionals/loops/exercise_00.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+for (let num = 10; num <= 40; num++) {
```
Comment (tictools) [inline challenge_03_loops_conditionals/loops-conditionals/loops/exercise_00.js]: Avoid mental mapping and abbr.

```suggestion
for (let number = 10; number <= 40; number++) {
```

⚠️ FIX: apply suggestion

File: challenge_03_loops_conditionals/loops-conditionals/loops/exercise_01.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+/*
+    1. Create a for loop that loops while index is less or equal than 10.
+    2. Each loop:
+        * create a new random number between 1 and 10 (hint: check documentation regarding Math.floor and Math.random methods)
+        * evaluate if random number is greater than 5
+        * if it is equal or greater than 5 print "Logged In!"
+        * if it is less than 5 print "Logged Out!"
+*/
+for (let i = 0; i <= 10; i++) {
```
Comment (tictools) [inline challenge_03_loops_conditionals/loops-conditionals/loops/exercise_01.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

This is alwaysthe output of your script
```
Logged In!
Logged In!
Logged In!
Logged In!
Logged In!
Logged In!
Logged In!
Logged In!
Logged In!
Logged In!
Logged In!
```

have you tested it before pushing? 🤔

⚠️ FIX: spot the 🐛 and fix it

File: challenge_03_loops_conditionals/loops-conditionals/loops/exercise_02.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    Checkpoint! 40
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+for (let i = 1; i <= 100; i++) {
```
Comment (tictools) [inline challenge_03_loops_conditionals/loops-conditionals/loops/exercise_02.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops_conditionals/loops-conditionals/while/exercise_00.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    1. Create a variable called "loggedIn" that starts off as false.
+    2. Create a while loop that loops while "loggedIn" is false.
+    3. Each loop, print out: "Incorrect login credentials"
+    3. After 3 loop iterations, change "loggedIn" to be true.
+    4. After the loops is done print out: "Sucessfully logged in!"
+
+    HINT1: Be careful for an infinite loop!
+*/
+let loggedIn = false;
+let i = 0;
```
Comment (tictools) [inline challenge_03_loops_conditionals/loops-conditionals/while/exercise_00.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops_conditionals/loops-conditionals/while/exercise_02.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+    Checkpoint! 40
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+let i = 1;
```
Comment (tictools) [inline challenge_03_loops_conditionals/loops-conditionals/while/exercise_02.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

**Review by tictools — CHANGES_REQUESTED** [review]
`node_modules` folder should never be pushed to remote repository.

In order to remove it, follow these steps:
* remove `node_modules` folder in your remote repository (branch `feat/challenge_03_loops-conditionals`)
* move to your local repo (same branch) and execute `git pull`
* fix requested changes and push them.


(same for `.DS_Store` file)

**Review by Manel0712 — APPROVED** [review]
Well Done!

**Review by tictools — APPROVED** [review]

**Pull request merged**